src/layers.js

```diff
@@ -0,0 +1,109 @@
+export const COLOR_RAMP = [
+  '#FFFFFF',
+  '#F0ECFF',
+  '#E2D9FF',
+  '#D4C6FF',
+  '#C5B4FF',
+  '#B7A1FF',
+  '#A98EFF',
+  '#9A7CFF',
+  '#8C69FF',
+  '#7E56FF',
+  '#7044FF'
+];
+
+const transparent = 'rgba(255, 255, 255, 0)';
+
+export const vectorLayerCarto = (iso, opacity = 1) => {
```

Comment:
This will have to change later because the app is going to have more than one vector layer.

Reply:
👍